green.py

```diff
@@ -1,8 +1,8 @@
 import Cimpl
-def green_channel():
+def 121_green_channel():
```

Comment:
simply green_channel() was fine for the function name.

the function name for the testing function is supposed to be **check_equal()**